src/main/java/com/segmentreader/mapreduce/AppContext.java

```diff
@@ -13,7 +14,18 @@
 
 public class AppContext {
```

Comment:
Please comment the purpose of this class and all inner classes as well in javadoc format.